.github/workflows/optimize-run-java-checks.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         uses: actions/setup-java@99b8673ff64fbf99d8d325f52d9a5bdedb8483e9 # v4
         with:
           distribution: "adopt"
-          java-version: "17"
+          java-version: "21"
```

Comment:
Does this mean we're changing our java version? Would that need documented?
https://docs.camunda.io/docs/reference/supported-environments/#component-requirements

Reply:
I thought I did it here time ago with this:
https://github.com/camunda/camunda-docs/commit/f56d15e7ef9d57853f0928bd456c82818d9031ac

I don't know why the web site is not

Reply:
ok all good then, just please make sure its updated